reflex/.templates/jinja/web/pages/base_page.js.jinja2

```diff
@@ -1,4 +1,5 @@
 {% import 'web/pages/utils.js.jinja2' as utils %}
+/** @jsxImportSource @emotion/react */
```

Comment:
Not sure why, but in my app this is rendering as

```javascript
/** @jsxImportSource @emotion/react */import { Fragment } from "react"
```

It still seems to work, somehow